domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/Response.java

```diff
@@ -33,6 +34,9 @@
 
     private final YesNoOption freeMediation;
 
+    @Size(max = 30, message = "may not be longer than {max} characters")
```

Comment:
would be good if we can validate that when freemediation is yes, mediationPhonenumber is provided